lib/garage/controller_helper.rb

```diff
@@ -34,7 +34,7 @@ def resource_owner_id
     # Use this method to render 'unauthorized'.
     # Garage user may overwrite this method to response custom unauthorized response.
     # @return [Hash]
-    def unauthorized_render_options
+    def unauthorized_render_options(error: nil)
```

Comment:
Thank you for your prompt response.

I think it could better to follow [the upstream](https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/rails/helpers.rb#L14) than using `*` as an argument
because doorkeeper 2.x does not have any arguments and we can still keep compatibility.